### PR TITLE
COMMERCE-11957

### PR DESCRIPTION
**Pull request by victorcdp**
https://liferay.atlassian.net/browse/COMMERCE-11957

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/edit_cp_specification_option.jsp [hunk 1]
```diff
@@ -53,4 +53,8 @@ else {
 			id="<%= CPSpecificationOptionFormNavigatorConstants.FORM_NAVIGATOR_ID_COMMERCE_PRODUCT_SPECIFICATION_OPTION %>"
 		/>
 	</div>
-</aui:form>
\ No newline at end of file
+</aui:form>
+
+<liferay-frontend:component
+	module="js/edit_cp_option_and_value"
+/>
```
Comment (brianikim) [inline modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/edit_cp_specification_option.jsp:60]: Why do we need this?

Comment (victorcdp) [reply modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/edit_cp_specification_option.jsp:60]: This js updates the key and removes slashes or dots in real time if the user tries inputting one, it's just a frontend validation, it's in the Options page as well.

**Comment by brianikim** [comment]
You can forward. Thanks.

**Pull request closed without merging**